packages/manager/modules/pci/src/projects/new/payment/index.js

```diff
@@ -29,6 +30,7 @@ angular
   .config(routing)
   .run(/* @ngTranslationsInject:json ./translations */)
   .component(component.name, component)
-  .service('pciProjectNewPayment', paymentService);
+  .service('pciProjectNewPayment', paymentService)
+  .service('pciProjectNew', projectNewservice);
```

Comment:
**nitpick:** The service name is not good enough. `new` until when? Please rename it with meaningful name.

Reply:
This request could be done in a separate ticket dedicated to the refactoring the name of this service. For now, I prefere to keep the name of this service as it's used in the manager.

Reply:
Can we have a ticket for refactoring then?